kafka-ui-react-app/src/components/Topics/List/List.tsx

```diff
@@ -41,25 +43,39 @@ const List: React.FC<Props> = ({
   const { isReadOnly } = React.useContext(ClusterContext);
   const { clusterName } = useParams<{ clusterName: ClusterName }>();
   const { page, perPage } = usePagination();
+  const [orderBy, setOrderBy] = React.useState<
+    TopicColumnsToSort | undefined
```

Comment:
It's better to use `null` instead of `undefined` for the state. That way we will know that we intentionally set it to `null` when we want to reset the value. In case of `undefined` it's not clear if we set it or it's broken somewhere.